www/profile/inner.js

```diff
@@ -503,6 +547,11 @@ define([
         if (!APP.editor) { return; }
         APP.editor.setValue(data.description || "");
         APP.editor.save();
+        if ($.trim(descriptionData) === "") {
```

Comment:
It's probably faster to use the native "trim" instead of jQuery: `descriptionData.trim()`